### ISSUE TITLE
icons issue

### ISSUE THREAD

**Issue by ZommbieWasTaken** [webbukkit/dynmap#3082]
On my dynmap I can't see coords on it, options to see things like worldborder or chunkmaster, unable to see players and world spawn stuff like that. Also every so often I get a sql js error. I will put a picture attached please he
![test23](https://user-images.githubusercontent.com/69566335/89974213-6fc5db00-dc30-11ea-98b1-8006b9e9d414.png)
lp.

**Comment by FedUpWith-Tech** [comment]
Hey to help get you support faster and so I know more about your specific issue can you please full out this info:

***

**Issue Description:** _Dynmap sample issue description. This description would include as much and as little detail necessary for us to understand the issue in its entirety._
* **Dynmap Version:** _dynmap-version_
* **Server Version:** _forge 1.16 build x, or paper 1.15.2 build x_
* **Pastebin of Configuration.txt:** _https://pastebin.com/5SETL3z2_
* **Server Host (if applicable):** _McProHosting, Shockbyte, Selfhosted, etc._
* **Pastebin of crashlogs or other relevant logs:** _https://pastebin.com/crashcausedbydynmap_
* **Other Relevant Data:** _I'm using this texture pack and these plugins and my map is from alpha .8_
* **Steps to Replicate:** _Please be clear in this section so we can replicate your issue_

**Comment by ZommbieWasTaken** [comment]
Dynmap version: v3.1-beta-2

Server Version is paper 1.15.2

Ima use mcpaste:  https://mcpaste.io/f273d0f586daa49c

Server Host: Pebblehost

Other data: None

**Comment by FedUpWith-Tech** [comment]
This is not an uncommon issue but unfortunately I haven't seen any solutions to this other than wait for the plugin to generate that file or don't use mysql. I'm not familiar enough with the codebase to even start to take a look to why this is happening, some people experience it but some don't. We need more data on this issue before we can replicate it and then when we can replicate this we will be able to start looking for the solution

**Comment by ZommbieWasTaken** [comment]
I just have my 1:500 scale of the map fully rendered no other configuration

**Comment by FedUpWith-Tech** [comment]
Assuming this is fixed in a later release or otherwise no longer relevant- please open a new issue if this persists.
